integration/tests/cook/test_basic.py

```diff
@@ -25,8 +25,8 @@ def setUp(self):
         util.wait_for_cook(self.cook_url)
 
     def test_basic_submit(self):
-        job_uuid, resp = util.submit_job(self.cook_url)
-        self.assertEqual(resp.status_code, 201)
+        job_uuid, resp = util.submit_job(self.cook_url, executor='cook')
```

Comment:
If the scheduler is configured to not use the Cook executor, it will still accept `cook` as an executor argument, but will end up using the Mesos command executor to execute the job. Do you want the scheduler to reject a `cook` job if the scheduler is not configured to use it?